### PR TITLE
Microprofile 13 config 12

### PR DESCRIPTION
**Pull request by juangon**

### CODE REVIEW

File: microprofile/microprofile-metrics-1.1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <groupId>io.thorntail.ts</groupId>
+        <artifactId>ts-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>ts-microprofile-metrics-1.1</artifactId>
+    <packaging>war</packaging>
+
+    <name>Swarm TS: MicroProfile Metrics 1.1</name>
```
Comment (Ladicek) [inline microprofile/microprofile-metrics-1.1/pom.xml]: Thorntail

File: microprofile/microprofile-openapi-1.0/src/main/java/org/wildfly/swarm/ts/microprofile/openapi/v10/ResponsePOJO.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package org.wildfly.swarm.ts.microprofile.openapi.v10;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+public class ResponsePOJO implements Serializable{
```
Comment (Ladicek) [inline microprofile/microprofile-openapi-1.0/src/main/java/org/wildfly/swarm/ts/microprofile/openapi/v10/ResponsePOJO.java]: Formatting.

File: microprofile/microprofile-opentracing-1.0/src/test/java/org/wildfly/swarm/ts/microprofile/opentracing/v10/MicroprofileOpentracing10IT.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    @Test
+    @InSequence(2)
+    @RunAsClient
+    public void trace() {
+        // the tracer inside the application doesn't send traces to the Jaeger server immediately,
+        // they are batched, so we need to wait a bit
+        await().atMost(5, TimeUnit.SECONDS).untilAsserted(() -> {
+            String response = Request.Get("http://localhost:16686/api/traces?service=test-traced-service").execute().returnContent().asString();
+            //System.out.println("---RESPONSE:-----" + response);
```
Comment (Ladicek) [inline microprofile/microprofile-opentracing-1.0/src/test/java/org/wildfly/swarm/ts/microprofile/opentracing/v10/MicroprofileOpentracing10IT.java]: Remove.

File: microprofile/microprofile-opentracing-1.0/src/test/java/org/wildfly/swarm/ts/microprofile/opentracing/v10/MicroprofileOpentracing10IT.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+@RunWith(Arquillian.class)
+public class MicroprofileOpentracing10IT {
+    
+    @Deployment(testable = false)
```
Comment (Ladicek) [inline microprofile/microprofile-opentracing-1.0/src/test/java/org/wildfly/swarm/ts/microprofile/opentracing/v10/MicroprofileOpentracing10IT.java]: Why not `@DefaultDeployment`?

**Comment by Ladicek** [comment]
If you can't split the changes in root `pom.xml` into individual commits (hint: it's pretty easy with `git add -p`), then better keep everything in one commit.

**Comment by juangon** [comment]
Fixed in https://github.com/rhoar-qe/thorntail-test-suite/pull/27. Closing...

**Comment by Ladicek** [comment]
FYI, I'm working on the fixes myself, should be able to merge later today or early tomorrow.

**Comment by Ladicek** [comment]
Ah.

**Comment by Ladicek** [comment]
Done in #27.

**Pull request closed without merging**